### PR TITLE
Fix YesNoFields

### PR DESCRIPTION
**Pull request by amponce**
### Summary

This is a fix to a number of fields on the final review page are showing blank or incorrectly. 

Related PR:  https://github.com/department-of-veterans-affairs/vets-website/pull/28893

### Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#83421

### Screenshot(s) 

| Vehicles| Required Fields| Review Page | 
| --- | --- | --- | 
|  |  |  |

## Tasks
- [x] Identify root cause of visual bug
- [x] Fix visual issue

## Acceptance criteria

- [x] Review page is displaying correctly 

## Testing
- [x] Run through FSR with the conditions listed below

Testing procedure:
To recreate: run through a full FSR with the following conditions:
- [x]  No spouse
- [x]  0 dependents
- [x]  No vehicles
- [x] No recreational vehicles
- [x]  No credit card bills
- [x]  No installment contracts

---

### CODE REVIEW

File: src/applications/financial-status-report/config/chapters/veteranInformationChapter.js
Comment (Dr-Pongo) [inline src/applications/financial-status-report/config/chapters/veteranInformationChapter.js:1]: Dependent's requirement still seems funky. I was able to continue w/o putting a value in

Comment (amponce) [reply src/applications/financial-status-report/config/chapters/veteranInformationChapter.js:1]: fixed

**Review by Dr-Pongo — DISMISSED** [review]
LGTM! 🚀

File: src/applications/financial-status-report/pages/expenses/repayments/installmentContracts.js [hunk 1]
```diff
@@ -13,16 +16,17 @@ export const uiSchema = {
...
   questions: {
-    hasRepayments: {
-      'ui:title':
-        'Do you make monthly payments on any installments contracts or other debts you make monthly payments on?',
-      'ui:webComponentField': YesNoField,
-      'ui:required': () => true,
-      'ui:errorMessages': {
+    hasRepayments: yesNoUI({
+      title:
+        'Do you make monthly payments on any installment contracts or other debts you make monthly payments on?',
```
Comment (josephrlee) [inline src/applications/financial-status-report/pages/expenses/repayments/installmentContracts.js:21]: Change this to say: **Do you make monthly payments on any installment contracts or other debts?**

**Review by josephrlee — CHANGES_REQUESTED** [review]
Just one minor copy edit.

**Review by josephrlee — APPROVED** [review]
LGTM! 🚀

**Review by Dr-Pongo — APPROVED** [review]
LGTM! 🚀

**Pull request merged**